conda/environments/all_cuda-115_arch-x86_64.yaml

```diff
@@ -1,24 +1,25 @@
-name: rmm_dev
+# This file was automatically generated by `rapids-dependency-file-generator`. Changes should not be made directly to this file.
```

Comment:
I wish these lines were wrapped shorter. This first line in particular is quite long. Maybe we can break at the period?
```suggestion
# This file was automatically generated by `rapids-dependency-file-generator`.
# Changes should not be made directly to this file.
```

Or shorten the entire message:
```
# This file is generated by `rapids-dependency-file-generator`.
# To make changes, edit ../../dependencies.yaml and run `rapids-dependency-file-generator`.
```

Reply:
I might even cut out the relative path and just write "edit dependencies.yaml".

Reply:
Addressed in https://github.com/rapidsai/dependency-file-generator/pull/5. I'll also need to update the GitHub Action here once that PR is merged.

Reply:
Done in https://github.com/rapidsai/shared-action-workflows/commit/e42de5845a804815162ba3e8f45d4e0bc8cf82b8.